### ISSUE TITLE
Questions regarding Examples

### ISSUE THREAD

**Issue by BradKML** [shankarpandala/lazypredict#386]
* Lazy Predict version: N/A
* Python version: N/A
* Operating System: N/A

### Description
Why does the current example not use a splitter function, instead relying on hard-coding?
Bonus Question: Are K-Fold and ShuffleSplit good ideas for increasing cross-testing?

Ref (documentation): https://lazypredict.readthedocs.io/en/latest/usage.html#regression
Ref (function): https://scikit-learn.org/stable/modules/classes.html#splitter-functions

**Comment by BrandonKMLee** [comment]
Sample Draft:
```python
from pandas import read_csv
from lazypredict.Supervised import LazyRegressor
from sklearn.model_selection import train_test_split
import numpy as np
df = read_csv('data_example.csv').dropna()
X, y = df.drop(['output'], axis=1).astype(np.float32), df['output'].astype(np.float32)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=13)
reg = LazyRegressor(verbose=0,ignore_warnings=False, custom_metric=None)
models,predictions = reg.fit(X_train, X_test, y_train, y_test)
```

**Comment by shankarpandala** [comment]
This question has been addressed in the issue comments. The examples use train_test_split which is the standard approach. Regarding K-Fold cross-validation, this has been added in PR #535 with the `cv` parameter: `LazyClassifier(cv=5)` for 5-fold cross-validation. The documentation will be updated to reflect these changes. Closing as answered.
